fluentd-elasticsearch/Gemfile

```diff
@@ -3,19 +3,19 @@
 
 source "https://rubygems.org"
 
-gem "fluentd", "1.12.3"
-gem "oj", "3.11.5"
+gem "fluentd", "1.13.3"
+gem "oj", "3.12.3"
 gem "fluent-plugin-multi-format-parser", "~> 1.0.0"
 gem "fluent-plugin-concat", "~> 2.5.0"
 gem "fluent-plugin-grok-parser", "~> 2.6.2"
-gem "fluent-plugin-prometheus", "~> 1.8.5"
+gem "fluent-plugin-prometheus", "~> 2.0.2"
```

Comment:
This looks like a major change. Did you ensure everything is fine?

Reply:
Tested across the e2e setup and everything seem to be working fine.  Plus there aren't any breaking changes between the releases. https://github.com/fluent/fluent-plugin-prometheus/compare/v1.8.5...v2.0.2